File: DataStructure_TA_PondS/Data Structure Algorithm/Source Code/Chapter 4/C/Node.cpp
#include <stdio.h>
#include <stdlib.h>
//structure Link List
struct Node{
  int item;
  struct Node *next;
};
//Function insert new node in Link List
struct Node *insertNode(int newItem,struct Node *nextNode){
  struct Node *newNode;
  newNode = (struct Node *)malloc(sizeof(struct Node));
  newNode->item = newItem;
  newNode->next = nextNode;
  return(newNode);
}

File: DataStructure_TA_PondS/CalProgram/section1/59051232/src/LinkedList/Node.java
package LinkedList;
public class Node{
   private Object item;
   Node next;
   public Node() {
      next = null;
   }
   public Node(Object newItem){
      item = newItem;
      next = null;
   }
   public Node(Object newItem,Node nextNode){
      item = newItem;
      next = nextNode;
   }//end constructor	
   //begin method
   public void setItem(Object newItem){
      item = newItem;
   }
   public Object getItem (){
      return item;
   }
   public void setNext(Node nextNode){
      next = nextNode;
   }
   public Node getNext() {
      return next;
   }
}
